selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/preferences/CheckErrorsWarningsTabTest.java

```diff
@@ -127,6 +141,13 @@ private void changeAllSettingsInErrorsWarningsTab(
     WaitUtils.sleepQuietly(DEFAULT_TIMEOUT);
   }
 
+  private String getTextInStringFromFile(String path) throws Exception {
```

Comment:
This method is partly duplicates [FileUtil.readFileToString(File file)](https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#readFileToString(java.io.File)).